functional_tests/playbooks/update.yml

```diff
@@ -0,0 +1,60 @@
+# ansible-playbook functional_tests/playbooks/update.yml -e "file_name=my_updated_file_name" -e "file_content='my updated file content'"
+
+---
+- hosts: localhost
+  connection: local
+  become: False
+  gather_facts: False
+  vars:
+    squest_token: 5a5554bdb3ad397721fe2a57e1a76a2df6279725
+    squest_api_url: http://127.0.0.1:8000/api
+    squest:
+      request:
+        instance:
+          id: 1
+          name: test
+          service: 1
+          spec: {
+            file_name: "my_file_name",
+            uuid_file: "20354797-185c-5fc1-8be9-22900a944807"
+          }
+          state: AVAILABLE
+
+  tasks:
+    - name: Get UUID
+      set_fact:
+        uuid_file: "{{ squest['request']['instance']['spec']['uuid_file'] }}"
+
+    - name: Generate path
+      set_fact:
+        file_path: "/tmp/squest_functional_test/{{ uuid_file }}"
+
+    - name: Prints variables
+      ansible.builtin.debug:
+        msg:
+          - "UUID: {{ uuid_file }}"
+          - "file_name: {{ file_name }}"
+          - "file_content: {{ file_content }}"
+
+    - name: Remove old file
+      ansible.builtin.file:
+        path: "{{ file_path }}/{{ squest['request']['instance']['spec']['file_name'] }}"
+        state: absent
```

Comment:
no needs to remove the file

Reply:
if file_name changes the old file will not be deleted